### PR TITLE
Change in anchor tag for navigating files. Fixes #6

### LINKED ISSUE #6: URL wrong
### Steps to reproduce:
1. Access maven url in browser (http://localhost/maven/snapshots/)
2. Click "org/"
3. Click "apache/"

The link should be
`http://localhost/maven/snapshots/org/apache/`
but linked me to
`http://localhost/maven/snapshots/orgapache/`

### PR DESCRIPTION
**Pull request by bharatsingh-tc**
Added "/" between the path and file name, only if the path is empty string to avoid successive forward slashes.

### CODE REVIEW

File: src/main/twirl/gitbucket/mavenrepository/files.scala.html [hunk 1]
```diff
@@ -38,12 +38,15 @@ <h1>
...
-                  <a href="@context.baseUrl/maven/@name/@path@file.getName/">@file.getName/</a>
-                } else {
+                }else{
                   <i class="octicon octicon-file-text"></i>
-                  <a href="@context.baseUrl/maven/@name/@path@file.getName">@file.getName</a>
+                }
+                @if(path.nonEmpty){
+                  <a href="@context.baseUrl/maven/@name/@path/@file.getName/">@file.getName/</a>
+                }else{
+                  <a href="@context.baseUrl/maven/@name/@path@file.getName/">@file.getName/</a>
```
Comment (takezoe) [inline src/main/twirl/gitbucket/mavenrepository/files.scala.html]: If `path` is empty, this line is able to be:
```html
<a href="@context.baseUrl/maven/@name/@file.getName/">@file.getName/</a>
```

**Review by takezoe — COMMENTED** [review]
@Bharat1103 Thanks for your pull request! I added a tiny comment. I hope you'll check it.

File: src/main/twirl/gitbucket/mavenrepository/files.scala.html (hunk 1 above)
Comment (bharatsingh-tc) [reply src/main/twirl/gitbucket/mavenrepository/files.scala.html]: Yea sure, I updated the code. Thanks

**Comment by takezoe** [comment]
@Bharat1103 Thanks! I released a new version: https://github.com/takezoe/gitbucket-maven-repository-plugin/releases/tag/1.3.1

**Pull request merged**